recipes/recipes_emscripten/pyb2d/recipe.yaml

```diff
@@ -0,0 +1,53 @@
+context:
+  version: 0.7.2
+  name: pyb2d
+
+package:
+  name: pyb2d
+  version: '{{ version }}'
+
+source:
+  - url: https://github.com/{{name}}/{{name}}/archive/refs/tags/{{ version }}.tar.gz
+    sha256: d7cd30da300ea21073f2e9cc715a61f8c26495ff8c57ebbb7109b4dcfe62fc7c
+  - path: CMakeLists.txt
+build:
+  number: 0
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - cmake
+    - ninja
+    - numpy
+  host:
+    - python
+    - pybind11
+    - numpy
+    - sqlite
```

Comment:
do you really need this?

Reply:
probably not , at least not. 
   - sqlite
    - zlib
    - bzip2
    - libffi
   Copy pasted residual...